pkgs/tools/admin/azure-cli/python-packages.nix

```diff
@@ -539,7 +539,8 @@ let
       websocket-client = super.websocket-client.overridePythonAttrs(oldAttrs: rec {
         version = "1.3.1";
 
-        src = oldAttrs.src.override {
+        src = super.fetchPypi {
+          inherit (oldAttrs) pname;
```

Comment:
other fix is use `hash = ` instead of `sha256 = ` but I favor homogenous syntax.